.github/workflows/Dockerfile.dev.yml

```diff
@@ -71,9 +76,9 @@ jobs:
     # save can.  Once buildx supports it we can update.
     # https://github.com/docker/buildx/issues/847
     - name: Build Quickstart Image
-      run: docker build -f Dockerfile.dev -t stellar/quickstart:dev . --build-arg STELLAR_CORE_IMAGE_REF=stellar-core --build-arg HORIZON_IMAGE_REF=stellar-horizon
+      run: docker build -f Dockerfile.dev -t $IMAGE --label org.opencontainers.image.revision="${{ env.HEAD_SHA }}" . --build-arg STELLAR_CORE_IMAGE_REF=stellar-core --build-arg HORIZON_IMAGE_REF=stellar-horizon
     - name: Save Quickstart Image
-      run: docker save stellar/quickstart:dev -o /tmp/image
+      run: docker save $IMAGE -o /tmp/image
     - name: Upload Quickstart Image
       uses: actions/upload-artifact@v2
       with:
```

Comment:
in the description it says disabling the `pubnet` network test, but it's still included below:
```
        network: [testnet, pubnet, standalone]
```

Reply:
actually for the secrets point it looks safe to me as used
![Screen Shot 2021-12-03 at 4 48 16 PM](https://user-images.githubusercontent.com/30449853/144690029-ece69b3b-fe88-4cbf-9d93-df1e2ab91da2.png)

Reply:
> 

I had removed pubnet, but then in #318 I managed to fix the pubnet build for dev so that it is no longer slow, so removing it is no longer necessary. I forgot to update the description after doing that.